docker/init.sh

```
@@ -1,14 +1,17 @@
 #!/bin/bash
 
+set -e
```

This ensures we fail fast if something goes wrong.